services/tables/src/main/java/com/linkedin/openhouse/tables/api/spec/v0/request/components/ReplicationConfig.java

```diff
@@ -0,0 +1,32 @@
+package com.linkedin.openhouse.tables.api.spec.v0.request.components;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Builder(toBuilder = true)
+@EqualsAndHashCode
+@Getter
+@AllArgsConstructor(access = AccessLevel.PROTECTED)
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class ReplicationConfig {
+  @Schema(description = "Replication destination cluster name", example = "clusterA")
+  @NotNull(
+      message =
+          "Incorrect destination specified. destination field for replication config cannot be null")
+  @Valid
+  String destination;
```

Comment:
Should we also have the destination cluster as ENUM? Would help if updates are required on it in future or we need to support multiple clusters. 
Validations can also check if a valid cluster is specified.

Reply:
What is the cluster convention here? Is OH cluster convention or gaas convention followed here from validation perspective? As this is OSS codebase we may not be able to define ENUM.

Reply:
Yes, there needs to be some validation on destination clusterName as it is coming from user end directly. Otherwise, a bad cluster name will only be caught while setting up the carbon job making the feed back loop difficult to handle. This being in OSS, will need some tests to identify how to implement.

Reply:
discussed offline, we can perform this validation on the LI codebase to validate the string against cluster definitions set there